packages/strapi/config/database.js

```diff
@@ -4,11 +4,16 @@ module.exports = ({ env }) => ({
     default: {
       connector: 'bookshelf',
       settings: {
-        client: 'sqlite',
-        filename: env('DATABASE_FILENAME', '.tmp/data.db'),
+        client: 'mysql',
+        host: env('DATABASE_HOST', '127.0.0.1'),
+        port: env.int('DATABASE_PORT', 3306),
+        database: env('DATABASE_NAME', 'strapi'),
+        username: env('DATABASE_USERNAME', 'root'),
+        password: env('DATABASE_PASSWORD', ''),
+        charset: 'utf8mb4',
```

Comment:
Thank you @wd0517 for updating the Strapi config for production deployment. 

Most of your changes look good to me. However, for the local development, I prefer using the file-based DB like SQLite because there is no additional DB and credentials need to be installed and initialized. I will tweak this setup a little and merge your code shortly.

Cheers.